pkgs/development/python-modules/pysonos/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, xmltodict
+, requests
+, ifaddr
+}:
+
+buildPythonPackage rec {
+  pname = "pysonos";
+  version = "0.0.12";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1pfm23v2mj8l878ilpy9xb0jbkpncnwyf4432haraswvf1cg2bxh";
+  };
+
+  propagatedBuildInputs = [ xmltodict requests ifaddr ];
+
+  # Need to override pytest-conv to an old version
+  doCheck = false;
```

Comment:
You mean pytest-cov? Just remove the version constraint using `substituteInPlace` in `postPatch`.

Reply:
of course,
`` pytest-cov>=2.4.0,<2.6 ```
in ``requirements-dev.txt`` file. Ok, I will try with 2.6.1 then.